Realm/RLMObjectSchema.mm

```diff
@@ -105,7 +91,7 @@ + (instancetype)schemaForObjectClass:(Class)objectClass {
     Class cls = objectClass;
     Class superClass = class_getSuperclass(cls);
     NSArray *props = @[];
-    while (superClass != RLMObjectBase.class) {
+    while (superClass && superClass != RLMObjectBase.class) {
```

Comment:
SwiftFakeObjectWithURL <- SwiftFakeObject <- NSObject

With this change, only `SwiftFakeObjectWithURL`'s properties will be considered, exactly as if it were a subclass of `RealmSwift.Object`